interface/resources/html/tabletHelp.html

```diff
@@ -77,13 +77,14 @@
             var handControllerImageURL = null;
             var index = 0;
             var count = 3;
-			var handControllerRefURL = "https://docs.highfidelity.com/explore/get-started/vr-controls.html#vr-controls";
-			var keyboardRefURL = "https://docs.highfidelity.com/explore/get-started/desktop.html#movement-controls";
-			var gamepadRefURL = "https://docs.highfidelity.com/explore/get-started/vr-controls.html#gamepad";
+            var handControllerRefURL = "https://docs.projectathena.dev/explore/get-started/vr-controls.html#vr-controls";
+            // Link below is currently throwing a 404
+			// var keyboardRefURL = "https://docs.projectathena.dev/get-started/desktop.html#movement-controls";
+			var gamepadRefURL = "https://docs.projectathena.dev/explore/get-started/vr-controls.html#gamepad";
```

Comment:
URL should be "https://docs.projectathena.dev/explore/get-started/desktop.html#movement-controls" (i.e., "explore" missing in commented-out URL.